bundles/org.openhab.binding.tibber/README.md

```diff
@@ -244,9 +247,8 @@ Number:ElectricCurrent     TibberAPILiveCurrent1                 "Live Current P
 Number:ElectricCurrent     TibberAPILiveCurrent2                 "Live Current Phase 2 [%.1 A]"              {channel="tibber:tibberapi:7cfae492:live_current2"}
 Number:ElectricCurrent     TibberAPILiveCurrent3                 "Live Current Phase 3 [%.1 A]"              {channel="tibber:tibberapi:7cfae492:live_current3"}
 Number:Power               TibberAPILivePowerProduction          "Live Power Production [%.0f W]"            {channel="tibber:tibberapi:7cfae492:live_powerProduction"}
+Number:Energy              TibberAPILiveAccumulatedProduction    "Accumulated Production [%.2f kWh]"         {channel="tibber:tibberapi:7cfae492:live_accumulatedProduction"}
+Number:Energy              TibberAPILiveLastMeterProduction      "Last Meter Production [%.2f kWh]"         {channel="tibber:tibberapi:7cfae492:live_lastMeterProduction"}
```

Comment:
Very minor. As all lines have a perfect layout, except this line lacks a space :-)

Reply:
Corrected.